styles.css

```diff
@@ -0,0 +1,43 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-size: 20px;
+  background-color: rgba(26, 25, 32, 0.991);
+  color: beige;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* background-color: #333; */
+
+  /* color: white; */
```

Comment:
- [ ] Removed unused code in your project for your code to look professional :smile: